contributors/devel/pull-requests.md

```diff
@@ -99,12 +100,15 @@ Now that your release notes are in shape, let's look at how the PR gets tested a
 
 ## The Testing and Merge Workflow
 
-The Kubernetes merge workflow uses comments to run tests and labels for merging PRs. NOTE: For pull requests that are in progress but not ready for review, prefix the PR title with `WIP` and track any remaining TODOs in a checklist in the pull request description.
+The Kubernetes merge workflow uses comments to run tests and labels for merging PRs.
+NOTE: For pull requests that are in progress but not ready for review,
+prefix the PR title with `WIP` or `[WIP]` and track any remaining TODOs
```

Comment:
>also I think it's [WIP] isn't it?

@cblecker we allow prefix with both `WIP` and `[WIP]`.

https://github.com/kubernetes/test-infra/blob/master/prow/plugins/wip/wip-label.go#L40